CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ project(libunwind)
 
 cmake_minimum_required(VERSION 3.16.1)
 
+set(CMAKE_C_STANDARD 11)
```

Comment:
This seems like a big policy/support move. Is there a process in libunwind for this?

Reply:
Perhaps rev the package version?

Reply:
This cmake is only used by the windows CI. everything else uses autoconf build system. This line fixes the CI build on Windows.

We are using C11 since https://github.com/libunwind/libunwind/commit/45fc222a00f239a16f1420c4446113e8a730b75a.